Standalone/Standalone/AdConfigurations.m

```diff
@@ -0,0 +1,48 @@
+//
+//  AdConfigurations.m
+//  Standalone
+//
+//  Copyright © 2020 Criteo. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#import "AdConfigurations.h"
+@import CriteoPublisherSdk;
```

Comment:
same than below, please use the new preferred way of importing the SDK:

```
#import <CriteoPublisherSdk/CriteoPublisherSdk.h>
```

Reply:
Changed to `#import <CriteoPublisherSdk/CriteoPublisherSdk.h>`